src/common/contexts/provider.tsx

```diff
@@ -2,25 +2,34 @@ import { MessageTitle } from "@govtechsg/tradetrust-ui-components";
 import { ethers, providers, Signer } from "ethers";
 import React, { createContext, FunctionComponent, useContext, useEffect, useState } from "react";
 import { INFURA_API_KEY, NETWORK_NAME } from "../../config";
-import { utils } from "@govtechsg/oa-verify";
+import { magic } from "./helpers";
+
+export enum SIGNER_TYPE {
+  IDENTITY = "Identity",
+  METAMASK = "Metamask",
+  MAGIC = "Magic",
+}
 
 const getProvider =
   NETWORK_NAME === "local"
     ? new providers.JsonRpcProvider()
-    : utils.generateProvider({ network: NETWORK_NAME, providerType: "infura", apiKey: INFURA_API_KEY });
+    : new ethers.providers.InfuraProvider(NETWORK_NAME, INFURA_API_KEY);
```

Comment:
no should use `generateProvider`, context -> https://github.com/Open-Attestation/adr/blob/master/provider.md

Reply:
ok changed back to generateProvider()